charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -98,3 +98,7 @@ serviceAccount:
     create: true
     name: ebs-snapshot-controller
     annotations: {}
+
+storageClasses: []
```

Comment:
It'd be really nice if we include the options here similar to the efs PR.

Reply:
You mean like an example? I do have an example storage class entry in comments just like the efs PR. Just that ebs doesn't really need much explicit config. I can provide a more detailed entry I suppose.

Reply:
I've added reference `parameters` now. Should we also mention other options like `volumeBindingMode` & `reclaimPolicy`?

Reply:
Sorry for the delay here. 

Yeah that would great. Ideally we should note if they're optional and default value if they have one.